### DISCUSSION TITLE
Email Verification Failed

### DISCUSSION THREAD

**Discussion by Wecodethis** [deta/cloud-docs#333] [General] [answered]
Hi, please help.. I signed up a while ago but havent been able to log in,  getting this error when verifying the link:

https://i.imgur.com/orfAEi5.png
![image](https://user-images.githubusercontent.com/36640821/180587253-e046d7e0-f632-41ff-90f5-a1122e31b326.png)

Thanks for your help!

**Comment by abdelhai** [comment, marked as answer]
hi, please send us an email to hello@deta.sh from the email you signed up with.
